app/login/events.py

```diff
@@ -33,8 +33,11 @@ def connect():
         log_event("join", current_user, room)
     db.session.commit()
 
+    for room in current_user.current_rooms:
+        socketio.emit("joined_room", dict(user=current_user.id, room=room.name), room=request.sid)
 
-@socketio.on('ready')
+
+# @socketio.on('ready')
```

Comment:
do you still need this?

Reply:
Weirdly I did not get a notification for your comment. 
I like to keep it for now until we are sure we really don't need this anymore.